### DISCUSSION TITLE
GUI doesn't show when start to simulate

### DISCUSSION THREAD

**Discussion by BoyuanTang331** [InteractiveComputerGraphics/SPlisHSPlasH#290] [Q&A]
when I try to generate more than one scene to simulate, I find this problem many times that the GUI doesn't show and wait for long time the code won't stop

![image](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH/assets/117408630/ff418f3e-a329-4d5c-b89e-071b8b838ccc)

why this problem happened and how can I avoid it?

Thanks

**Comment by janbender** [comment]
What exactly do you mean by "generate more than one scene"?

**Reply by BoyuanTang331** [reply]
just like I open two custom_scense.py and running these two file at same time, use two file to simulate.

I deleted the repo and downloaded it once again it will solved before, I guess it may caused by the cache problem. It is very weird

This time I wrote this new code on a scratch file, at beginning it can run and simulate and also show the GUI window correctly, but once I changed some parameters it comes out this problem

I move this scratch file to the repo dictionary, the code runs temporal

**Reply by janbender** [reply]
Can you try to run it without using cached files? There is a command line option "--no-cache".
